composer.json

```diff
@@ -13,9 +13,9 @@
         "nesbot/carbon": "^2.38",
         "ratchet/pawl": "^0.4.1",
         "react/datagram": "1.5.*",
-        "symfony/options-resolver": "^5.1.3",
+        "symfony/options-resolver": "^5.1.3 || ^6.0",
         "trafficcophp/bytebuffer": "^0.3",
-        "monolog/monolog": "^2.1",
+        "monolog/monolog": "^2.1 || ^3.1",
```

Comment:
Any particular reason requiring 3.1 instead of 3.0?

Reply:
```suggestion
        "monolog/monolog": "^2.1 || ^3.0",
```

Reply:
Nope, it should be no issue.